site/conferences/2022-render-atl

```diff
@@ -0,0 +1,11 @@
+---
+title: Render-Atlanta (RenderATL) 2022
+url: https://www.renderatl.com/
+cocUrl: https://www.renderatl.com/code-of-conduct
```

Comment:
This is a 404 page. Have y'all established one yet?

Reply:
Hey Chris! 

It's been updated and [can be viewed here](https://www.renderatl.com/thecode#/)